tests/transform/test_model_details.py

```diff
@@ -130,6 +139,7 @@ class TestTransformModelDetails:
                 "allele_info": "model_details_allele_info_good_test_input.csv",
                 "model_info": "model_details_model_info_missing_column_input.csv",
                 "pathology": "model_details_pathology_good_test_input.csv",
+                "model_results_info": "model_details_model_results_info_good_test_input.csv",
```

Comment:
Could you add a test for when the model details file _doesn't_ contain the model referenced by the rest of the files? This actually seems likely to happen: if we add data for a new model and accidentally run this before updating the model details file.

Reply:
I think your question is:
If we are building `model_details` for a model that isn't flagged to include any of the three links, should we throw an error?

We will sometimes have models with only pathology and biomarker data (which aren't represented by links here), and we may also start introducing 'emerging models' that do not yet have results of any type at all.

Since there are valid cases where all three of these fields will be null, I don't think throwing an error would be helpful.

Reply:
There are two different model_details input files:

* model_info (https://www.synapse.org/Synapse:syn61378590)
Specifies general model metadata values like name, jax_id, and so on. If a model does not have a row in model_info, we should not be trying to generate a model_details object for it.

* model_results_info (https://www.synapse.org/Synapse:syn68377668)
Specifies which result types we have for a model. If a model does not have a row in model_results_info, we could either:
1. infer that there are no results for that model (yet) available 
2. throw an error

I'm not totally against 2, it would just mean that we'd need to add an empty row to model_results_info when we add an emerging model.